data/hai/hai jobs.txt

```diff
@@ -298,205 +298,6 @@ mission "Hai Festival [3]"
 
 
 
-phrase "unfettered aid payment dialog"
-	word
-		`You drop off the aid (or as they like to refer to it, "tribute") for the Unfettered, and receive <payment> from your contact on <origin>.`
-
-phrase "unfettered aid pickup dialog"
-	word
-		`The Hai dockworkers load the food for the Unfettered onto your ship. One of them says to you,`
-		`As the Hai load the food onto your ship, one of them says,`
-	word
-		` `
-	word
-		`"Thank you. This is far better than watching our kinfolk starve."`
-		`"Tell our brothers and sisters that if they will only repent, they will be welcome to come home."`
-
-mission "Unfettered Aid [0]"
-	name "Aid shipment to Unfettered"
-	job
-	repeat
-	clearance "After speaking with the Unfettered and explaining your presence, they give you permission to land with the cargo."
-	description "Deliver <tons> of food to the Unfettered on <destination>. You will be paid <payment>."
-	cargo "food (aid)" 25 2 .05
-	to offer
-		random < 40
-		"cargo space" >= 50
-		has "Ask the Hai about the Unfettered: offered"
-	source
-		government "Hai"
```

Comment:
This applies to all missions of this type.

Jobs/missions which offer in Hai space, should be in Hai files, even if they're related to the Unfettered.

Unfettered files should be for jobs or missions that offer in Unfettered space, or if such missions were to exist - potentially from Unfettered agents in Hai space. 

These aid missions are *Hai* missions.